kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/status/modem/Bearer.java

```diff
@@ -26,17 +27,17 @@ public class Bearer {
     private final String name;
     private final boolean connected;
     private final String apn;
-    private final BearerIpType ipType;
+    private final Set<BearerIpType> ipTypes;
```

Comment:
This code was autogenerated on my RPi with version 1.14.12. Why did you need to change it?

There are two possible causes for this and one scares me:
1. The autogen script fumbled and the code was not correct
2. You used another MM version for which the signature of these methods is different <- This might be an issue